examples/android/android-studio/Cukeulator/app/build.gradle

```diff
@@ -33,9 +47,188 @@ android {
     }
 }
 
+// ==================================================================
+// Project dependencies
+// ==================================================================
+
 dependencies {
-    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
-    androidTestCompile 'com.android.support.test:testing-support-lib:0.1'
+    androidTestCompile 'com.android.support.test.espresso:espresso-core:3.0.1'
+
+    // Use the stable Cucumber version
     androidTestCompile 'io.cucumber:cucumber-android:2.3.0'
+    //
+    // The following dependency works, if you build Cucumber-JVM on your local machine:
```

Comment:
Should be added to the android README instead of in comments here?

Reply:
@mlvandijk Thanks for your feedback

Please look at the end of [README.md](/cgnuechtel/cucumber-jvm/tree/master/examples/android/android-studio/Cukeulator/README.md)

There is a reference to build.gradle. I think it is better to leave the technical details in build.gradle since it does not affect most users.